integration/helper_test.go

```diff
@@ -263,6 +263,78 @@ options:
 			Expect(err).ToNot(HaveOccurred())
 			assertNoErrors()
 		})
+		It("gpbackup_helper will not error out when plugin writes something to stderr", func() {
+			setupRestoreFiles("", true)
+
+			err := exec.Command("touch", "/tmp/GPBACKUP_PLUGIN_LOG_TO_STDERR").Run()
+			Expect(err).ToNot(HaveOccurred())
+
+			go func() {
+				for _, i := range []int{1, 3} {
+					contents, _ := ioutil.ReadFile(fmt.Sprintf("%s_%d_0", pipeFile, i))
+					Expect(string(contents)).To(Equal("here is some data\n"))
+				}
+			}()
+
+			args := []string{
+				"--toc-file", tocFile,
+				"--oid-file", restoreOidFile,
+				"--pipe-file", pipeFile,
+				"--content", "1",
+				"--single-data-file",
+				"--restore-agent",
+				"--data-file", dataFileFullPath,
+				"--plugin-config", examplePluginTestConfig}
+			helperCmd := exec.Command(gpbackupHelperPath, args...)
+
+			output, err := helperCmd.CombinedOutput()
+			printHelperLogOnError(err)
+			Expect(err).ToNot(HaveOccurred())
+
+			outputStr := string(output)
+			Expect(outputStr).To(ContainSubstring("Some plugin warning"))
+
+			err = exec.Command("rm", "/tmp/GPBACKUP_PLUGIN_LOG_TO_STDERR").Run()
+			Expect(err).ToNot(HaveOccurred())
+
+			assertNoErrors()
+		})
```

Comment:
this test covers only first using logPlugin in defer func

Reply:
new test was added